functions/src/utils.js

```diff
@@ -148,5 +149,75 @@ exports.typesenseDocumentFromSnapshot = async (firestoreDocumentSnapshot, fields
 
   typesenseDocument.id = firestoreDocumentSnapshot.id;
 
+  Object.entries(contextParams).forEach(([key, value]) => {
```

Comment:
Could you add a length check for the array here to always check that it's not just an empty object?

Reply:
sounds good

Reply:
The change doesn't seem to appear here. It could be because of amending / force pushing afterwards